doc/manual/install-judgehost.rst

```diff
@@ -76,23 +72,24 @@ paths, be sure to update the sudoers rules accordingly.
 Creating a chroot environment
 -----------------------------
 
-The judgedaemon executes submissions inside a chroot environment for
-security reasons. By default it mounts parts of a prebuilt chroot tree
-read-only during this judging process (using the script
-``lib/judge/chroot-startstop.sh``). This is needed to support
-extra languages that require access to interpreters or support
+The judgedaemon compiles and executes submissions inside a chroot
+environment for securwty reasons. By default it mounts parts of a
+prebuilt chroot tree read-only during this judging process (using
+the script ``lib/judge/chroot-startstop.sh``). This is needed to
+support languages that require access to interpreters or support
```

Comment:
This should be rewritten as it still seems to imply that a chroot environment is only needed for interpreted languages, while now it's going to be needed for any language, compiled or interpreted (or anything in between, like Java).